mti_nma/steps/singlecell/singlecell.py

```diff
@@ -69,69 +73,68 @@ def run(self, cell_line_id='AICS-13', nsamples=16, **kwargs):
             Number of cells to sample randomly (but with set seed) from dataset
         '''
 
-        import uuid
-        import numpy as np
         np.random.seed(666)
-        import pandas as pd
-        from tqdm import tqdm
-        from pathlib import Path
-        from aicsimageio import AICSImage, writers
 
         if nsamples > 0:
 
             self.manifest = pd.DataFrame([])
 
+            # Load data from labkey and store in local dataframe
             print("Loading data from LabKey...")
-
-            df = query_data_from_labkey(cell_line_id=cell_line_id) # 13 = Lamin
-
-            print(f"Number of FOVs available = {df.shape[0]}. Sampling {nsamples} FOVs now.")
-
+            df = query_data_from_labkey(cell_line_id=cell_line_id)  # 13 = Lamin
+            print(
+                f"Number of FOVs available = {df.shape[0]}."
+                f"Sampling {nsamples} FOVs now.")
             df = df.sample(n=nsamples)
 
+            # Process each FOV in dataframe
             for fov_id in tqdm(df.index):
 
                 sx = df.PixelScaleX[fov_id]
                 sy = df.PixelScaleY[fov_id]
                 sz = df.PixelScaleZ[fov_id]
 
-                # C=-2 may not return the DNA channel for other cell lines. Need validation.
-                raw = AICSImage(df.ReadPathRaw[fov_id]).get_image_data('ZYX', S=0, T=0, C=-2)
-                seg = AICSImage(df.ReadPathSeg[fov_id]).get_image_data('ZYX', S=0, T=0, C= 0)
+                # C=-2 may not return the DNA channel for other cell lines.
+                # Need validation.
+                raw = AICSImage(
```

Comment:
If this is the latest version of aicsimageio you should see a speed up if (and when) you switch over the `bin/all.py` to be distributed with prefect / dask :)